pkg/bootstrap/option/convert.go

```diff
@@ -170,6 +171,17 @@ func addressConverter(addr string) convertFunc {
 		if err != nil {
 			return nil, fmt.Errorf("unable to parse %s address %q: %v", o.name, addr, err)
 		}
+		if host == "$(HOST_IP)" {
```

Comment:
Do we care about other variables? for simplicity I like just host ip, just curious if there are others.

Reply:
I searched manifests folder for $(.*), there are only POD_NAMESPACE and HOST_IP, so with this change I think we should be good at least for options shipped with Istio? Or can you think of some other cases?